frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -85,15 +91,32 @@ export const experimentLogic = kea<experimentLogicType>({
         ],
     },
     listeners: ({ values, actions }) => ({
-        createExperiment: async ({ draft }) => {
+        createExperiment: async ({ draft, runningTime, sampleSize }) => {
+            let response: Experiment | null = null
+            const isUpdate = !!values.newExperimentData?.id
+
             try {
                 if (values.newExperimentData?.id) {
-                    await api.update(`api/projects/${values.currentTeamId}/experiments/${values.experimentId}`, {
-                        start_date: dayjs(),
-                    })
+                    response = await api.update(
```

Comment:
this flow now allows for editing draft experiments (once we get to adding the Edit button)